README.md

```diff
@@ -2,15 +2,31 @@
 Maintain SKU specific package content such as templates and other static files.
 
 # Quick Start
-* Build SKU package
+* Prerequisites
+
+  * Debian based build system or equivalent that can install the necessary dependencies (i.e. Ubuntu 12.04 & up)
+  * Install the following packages (Ubuntu example):
+  
+   ```
+   sudo apt-get install dh-make devscripts debhelper
+   ```
 
-```
-$ ./build-package.bash <sku_pack_directory> <subname>
-```
+* Build SKU package
 
+  ```
+  $ ./build-package.bash <sku_pack_directory> <subname>
+  ```
+  
+  * Example
+  
+  ```
+  $ ./build-package.bash quanta-t41 master
+  ```
+  
 * Register a new SKU with the package
-```
-$ curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack
-```
+
+  ```
+  $ curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack
```

Comment:
@brianparry I just noticed that this is still referencing /common.  Were we changing references to explicitly say 2.0?